### PR TITLE
Ports: Add editline 1.17.1

### PR DESCRIPTION
**Pull request by emilytrau**
https://github.com/troglobit/editline

### CODE REVIEW

File: Ports/editline/package.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+
+port="editline"
```
Comment (GMTA) [inline Ports/editline/package.sh:3]: Please use single quotes for each non-interpolated string (i.e. `'editline'`)

File: Ports/editline/package.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+version="1.17.1"
+useconfigure="true"
+use_fresh_config_sub=true
```
Comment (GMTA) [inline Ports/editline/package.sh:6]: `true` here is also just a string; `'true'`

Comment (emilytrau) [reply Ports/editline/package.sh:6]: Sorry I misread the commend. Should be good now :+1:

Comment (linusg) [reply Ports/editline/package.sh:6]:
```suggestion
use_fresh_config_sub='true'
```

Comment (GMTA) [reply Ports/editline/package.sh]: @emilytrau Still missing `use_fresh_config_sub='true'`...

**Review by GMTA — APPROVED** [review]

**Pull request merged**